cmd/traffic/cmd/manager/internal/cliconfig/cliconfig.go

```diff
@@ -3,98 +3,95 @@ package cliconfig
 import (
 	"context"
 	"fmt"
-	"io"
-	"os"
-	"path"
-	"path/filepath"
 	"sync"
 
-	"github.com/fsnotify/fsnotify"
+	core "k8s.io/api/core/v1"
+	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/watch"
 
 	"github.com/datawire/dlib/dlog"
+	"github.com/telepresenceio/telepresence/v2/pkg/k8sapi"
 )
 
-const cfgFileName = "client.json"
+const (
+	cfgFileName      = "client.yaml"
```

Comment:
should we accept both json and yaml going forward if json is something we used to support?

Reply:
Backward compatibility is not an issue in this case, because this whole thing is brand new. The configmap doesn't exist in 2.8.5.